spec/javascripts/health-care/components/form-elements/ErrorableRadioButtonGroup.spec.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import ReactTestUtils from 'react-addons-test-utils';
+import SkinDeep from 'skin-deep';
+
+import ErrorableRadioButtonGroup from '../../../../../_health-care/_js/components/form-elements/ErrorableRadioButtonGroup';
+
+describe('<ErrorableRadioButtonGroup>', () => {
+  const options = [{ value: 1, label: 'first' }, { value: 2, label: 'second' }];
+
+  describe('propTypes', () => {
+    let consoleStub;
+    beforeEach(() => {
+      consoleStub = sinon.stub(console, 'error');
+    });
+
+    afterEach(() => {
+      consoleStub.restore();
+    });
+
+    it('label is required', () => {
+      SkinDeep.shallowRender(
+        <ErrorableRadioButtonGroup options={options} onValueChange={(_update) => {}}/>);
+      sinon.assert.calledWithMatch(consoleStub, /Required prop `label` was not specified in `ErrorableRadioButtonGroup`/);
+    });
+
+    it('label must be a string', () => {
+      SkinDeep.shallowRender(
+        <ErrorableRadioButtonGroup label options={options} onValueChange={(_update) => {}}/>);
+      sinon.assert.calledWithMatch(consoleStub, /Invalid prop `label` of type `boolean` supplied to `ErrorableRadioButtonGroup`, expected `string`./);
+    });
+
+    it('onValueChange is required', () => {
+      SkinDeep.shallowRender(<ErrorableRadioButtonGroup label="test" options={options}/>);
+      sinon.assert.calledWithMatch(consoleStub, /Required prop `onValueChange` was not specified in `ErrorableRadioButtonGroup`/);
+    });
+
+    it('onValueChange must be a function', () => {
+      SkinDeep.shallowRender(<ErrorableRadioButtonGroup label="test" options={options} onValueChange/>);
+      sinon.assert.calledWithMatch(consoleStub, /Invalid prop `onValueChange` of type `boolean` supplied to `ErrorableRadioButtonGroup`, expected `function`/);
+    });
+
+    it('options is required', () => {
+      SkinDeep.shallowRender(
+        <ErrorableRadioButtonGroup label="test" onValueChange={(_update) => {}}/>);
+      sinon.assert.calledWithMatch(consoleStub, /Required prop `options` was not specified in `ErrorableRadioButtonGroup`/);
+    });
+
+    it('options must be an object', () => {
+      SkinDeep.shallowRender(
+        <ErrorableRadioButtonGroup label="test" options onValueChange={(_update) => {}}/>);
+      sinon.assert.calledWithMatch(consoleStub, /Invalid prop `options` of type `boolean` supplied to `ErrorableRadioButtonGroup`, expected an array/);
+    });
+  });
+
+  xit('ensure value changes propagate', () => {
```

Comment:
Let's try to get these tests that are commented out passing before we merge this.